src/Swift.Bindings/src/Program.cs

```diff
@@ -182,11 +182,12 @@ public static void GenerateBindings(Queue<string> paths, string outputDirectory,
             }
 
```

Comment:
You might want:
```csharp
var libraryDirectory = Path.Combine(Directory.GetParent(outputDirectory)!.FullName, "Library");
Directory.CreateDirectory(libraryDirectory);
...
var destFilePath = Path.Combine(libraryDirectory, fileName);
```
Factoring it out eliminates redundant string allocation/concatenation.

Also, if you do this:
```
var srcDirInfo = new DirectoryInfo(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Library"));
foreach (var fileInfo in srcDirInfo.GetFiles()) { // returns FileInfo
    file.CopyTo(Path.Combine(libraryDirectory, file.Name));
}
```
It's a little tighter.